cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -239,6 +241,12 @@ func main() {
 		"Whether to use the legacy gke_container MonitoredResource type instead of k8s_container").
 		Default("false").BoolVar(&cfg.useGkeResource)
 
+	a.Flag("stackdriver.store-in-files", "Whethere to store payload in filesystem. For use in troubleshooting.").
+		Default("false").BoolVar(&cfg.storeInFiles)
+
+	a.Flag("stackdriver.store-in-files-directory", "Directory to store payload. For use in troubleshooting.").
```

Comment:
"the payload" is vague. Please document more clearly:
- what payload this is
- whether the requests will still be sent to Stackdriver.

I would remove "for use in troubleshooting", as it doesn't add much information but takes up space.